### ISSUE TITLE
Unconditional Image Generation with the provided checkpoint

### ISSUE THREAD

**Issue by basusattwik** [alexanderswerdlow/unidisc#5]
Hello: 
I would like to use UniDisc as a discrete diffusion model to generate images without any conditioning via text. 

1. Would the provided checkpoints support good image quality for unconditional generation (i.e., with appropriate code changes)?   
2. Is the model capable of generating images at 256x256 size? 

By any chance are there pretrained checkpoints that you could provide for this? Thanks!

**Comment by alexanderswerdlow** [comment]
Hi!

1. I haven't tested this directly with the checkpoint but it should be supported as it was trained with dropout
2. It should support this

These should be supported (although not directly tested) with the provided checkpoint in the readme.

Thanks!
